### DISCUSSION TITLE
How to create catch-all instead of 404 error page

### DISCUSSION THREAD

**Discussion by hlorofos** [statamic/cms#7266] [Q&A]
I'm trying to create a catch all page which would work like this

`Route::statamic('{q}', 'landing', ['layout' => 'custom_search']);`

this is my last line in `web.php` file, however this catch all getting higher priority and neither control panel nor existing pages showing up.

Is there a possibility to return back custom catch-all view with 200 return code?

**Comment by edalzell** [comment]
set your 404 page to do the search?
